### ISSUE TITLE
Colors for selected, unselected states on RangeSlider have poor contrast

### ISSUE THREAD

**Issue by ssciolla** [tl-its-umich-edu/my-learning-analytics#805]
Thank you for contributing to this project!

- Make sure to search the [issues](https://github.com/tl-its-umich-edu/my-learning-analytics/issues) for duplicates first!

## Expected behavior (A clear and concise description of what you expected to happen) : 

There should be a high enough contrast ratio (WCAG AA, 4.5 to 1) between the selected and un-selected portions of the `RangeSlider` component in the Resources Accessed View so that more users can clearly determine what week range is selected.

## Describe the bug (Tell us what happens instead of the expected behavior) :

From 10/28/19 accessibility review:

"Selected and unselected slider segments colors are very subtle. Contrast ratio between selected and unselected is 1.2:1. ... Some users may have difficulty seeing what weeks are selected."

**Comment by ssciolla** [comment]
This will likely be addressed by the changes in PR #775, but I'll confirm the contrast ratio meets the mid-range level.

**Comment by jennlove-um** [comment]
The contrast between the blue (selected) and gray (unselected) on the range slider have been changed.  They meet the 3:1 contrast required for graphical elements and user interface components.

**Comment by jennlove-um** [comment]
Testing passes.

**Comment by gsilver** [comment]
@jennlove-um - excellent.
